src/compat/eme/custom_media_keys/index.ts

```diff
@@ -51,6 +53,41 @@ let requestMediaKeySystemAccess : null |
                                                     CustomMediaKeySystemAccess>)
                                 = null;
 
+let _setMediaKeys : ((elt: HTMLMediaElement,
+                      mediaKeys: MediaKeys | ICustomMediaKeys | null) => void) =
+  function defaultSetMediaKeys(elt: HTMLMediaElement,
+                               mediaKeys: MediaKeys | ICustomMediaKeys | null) {
+    if (mediaKeys !== null && isCustomMediaKeys(mediaKeys)) {
```

Comment:
That condition seems a little random here.

Let's just not care, no?

The default implementation of `setMediaKeys` is to call the default `setMediaKeys` if it exists and if not, the vendored version.
Checking if mediaKeys is null or not feel out of place in that function to me.